UnitTests/GravatarTests/DirectoryImageCacheTests.cs

```diff
@@ -176,9 +166,9 @@ public void Clear_should_ignore_errors()
         [TestCase(null)]
         [TestCase("")]
         [TestCase("\t")]
-        public async Task DeleteImage_should_exit_if_filename_not_supplied(string fileName)
+        public void DeleteImage_should_throw_if_filename_not_supplied(string fileName)
         {
-            await _cache.DeleteImageAsync(fileName);
+            Assert.ThrowsAsync<ArgumentException>(async () => await _cache.DeleteImageAsync(fileName));
```

Comment:
:bulb: The `async` and `await` are unnecessary here. This occurs in a few other cases where the test delegate simply passes through the result of an asynchronous operation.

Reply:
Sure. I've seen this in a bit of GE's non-test code and thought it was just a style thing. It's a bit wasteful.